src/Symfony/Component/PropertyInfo/Extractor/ConstructorExtractor.php

```diff
@@ -28,15 +30,22 @@ public function __construct(
     ) {
     }
 
-    public function getTypes(string $class, string $property, array $context = []): ?array
+    public function getType(string $class, string $property, array $context = []): ?Type
     {
         foreach ($this->extractors as $extractor) {
-            $value = $extractor->getTypesFromConstructor($class, $property);
+            $value = $extractor->getTypeFromConstructor($class, $property);
             if (null !== $value) {
                 return $value;
             }
         }
 
         return null;
     }
+
+    public function getTypes(string $class, string $property, array $context = []): ?array
+    {
+        trigger_deprecation('symfony/property-info', '7.1', 'The "%s()" method is deprecated. Use "%s::getType()" instead.', __METHOD__, self::class);
```

Comment:
```suggestion
        trigger_deprecation('symfony/property-info', '7.1', 'The "%s()" method is deprecated, use "%s::getType()" instead.', __METHOD__, self::class);
```
Same for all notices